### PR TITLE
Fix: no application add create application link

### LINKED ISSUE #301: Addon a link of create application if no application exist
image

### PR DESCRIPTION
**Pull request by wangbow**
Signed-off-by: wb-wb665667 <wb-wb665667@alibaba-inc.com>

### Description of your changes
Fixes #301 no application add create application link

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary.
- [x] Run `yarn lint` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### Special notes for your reviewer
image

### CODE REVIEW

File: src/pages/ApplicationList/components/CardContent/index.tsx [hunk 1]
```diff
@@ -50,11 +51,27 @@ class CardContent extends React.Component<Props, State> {
...
+              <Translation>There is no applications</Translation>
+              <main>
+                <Button
+                  component="a"
+                  ghost={true}
+                  onClick={() => {
+                    setVisible(true);
+                  }}
+                >
+                  {' '}
```
Comment (barnettZQG) [inline src/pages/ApplicationList/components/CardContent/index.tsx:69]: Please remove the blank.

File: src/pages/ApplicationList/components/CardContent/index.tsx [hunk 2]
```diff
@@ -50,11 +51,27 @@ class CardContent extends React.Component<Props, State> {
...
+                  Please click create applications{' '}
```
Comment (barnettZQG) [inline src/pages/ApplicationList/components/CardContent/index.tsx:70]:
```suggestion
                  Creating first application
```

File: src/pages/ApplicationList/components/CardContent/index.tsx (hunk 1 above)
Comment (wangbow) [reply src/pages/ApplicationList/components/CardContent/index.tsx]: done

File: src/pages/ApplicationList/components/CardContent/index.tsx (hunk 2 above)
Comment (wangbow) [reply src/pages/ApplicationList/components/CardContent/index.tsx]: done

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**